pages/utils/graph_utils.py

```diff
@@ -25,14 +36,10 @@ def get_graph_time_values(interval):
         x_name = "Week"
         hover = "Week: %{x|%b %d, %Y}"
         period = "W"
-    elif interval == "M":  # if statement for months
-        x_r = [str(today - dt.timedelta(weeks=104)), str(today)]
-        x_name = "Month"
-        hover = "Month: %{x|%b %Y}"
-        period = "M"
-    elif interval == "M1":  # if statement for months
+    elif interval == "M" or interval == "M1":  # if statement for months
```

Comment:
same here

Reply:
(same as above)
haven't tested it, I'll check to see if it seems to comply on my end.